vnext/ReactUWP/Views/ScrollViewManager.cpp

```diff
@@ -206,32 +318,91 @@ void ScrollViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate, folly::dy
 
         scrollViewer.HorizontalScrollMode(horiz ? winrt::ScrollMode::Auto : winrt::ScrollMode::Disabled);
         scrollViewer.VerticalScrollMode(horiz ? winrt::ScrollMode::Disabled : winrt::ScrollMode::Auto);
+        ScrollViewUWPImplementation(scrollViewer).SetHorizontal(horiz);
       }
     }
     else if (pair.first == "showsHorizontalScrollIndicator")
     {
       if (pair.second.isBool())
+      {
         scrollViewer.HorizontalScrollBarVisibility(pair.second.getBool() ? winrt::ScrollBarVisibility::Visible : winrt::ScrollBarVisibility::Hidden);
+      }
     }
     else if (pair.first == "showsVerticalScrollIndicator")
     {
       if (pair.second.isBool())
+      {
         scrollViewer.VerticalScrollBarVisibility(pair.second.getBool() ? winrt::ScrollBarVisibility::Visible : winrt::ScrollBarVisibility::Hidden);
+      }
     }
     else if (pair.first == "minimumZoomScale")
     {
       if (pair.second.isNumber())
+      {
         scrollViewer.MinZoomFactor(static_cast<float>(pair.second.asDouble()));
+      }
     }
     else if (pair.first == "maximumZoomScale")
     {
       if (pair.second.isNumber())
+      {
         scrollViewer.MaxZoomFactor(static_cast<float>(pair.second.asDouble()));
+      }
     }
     else if (pair.first == "zoomScale")
     {
       if (pair.second.isNumber())
+      {
         scrollViewer.ChangeView(nullptr, nullptr, static_cast<float>(pair.second.asDouble()));
+      }
+    }
+    else if (pair.first == "snapToInterval")
+    {
+      if (pair.second.isNumber())
+      {
+        ScrollViewUWPImplementation(scrollViewer).SnapToInterval(pair.second.asDouble());
+      }
+    }
+    else if (pair.first == "snapToOffsets")
+    {
+      if (pair.second.isArray())
+      {
+        auto vector = [offsets = pair.second]()
+        {
+          auto vector = winrt::single_threaded_vector<float>();
```

Comment:
can we rename one of these 'vectors' to something else.  I guess the lambda doesn't generate a warning about variable hiding but still odd to have 2 variables with the same name so close together